configure.ac

```diff
@@ -63,6 +63,12 @@ AC_ARG_WITH([stage1-flavors],
             [RKT_STAGE1_FLAVORS="${withval}"],
             [RKT_STAGE1_FLAVORS=auto])
 
+AC_ARG_WITH([stage1-flavors-version-override],
+            [AS_HELP_STRING([--with-stage1-flavors-version-override],
+                            [override version of stage1 flavor taken from rkt version with custom one; it also changes default stage1 version in flavor setup])],
```

Comment:
override the version of stage1 with a custom one; it also changes the default stage1 version in flavor setup